asv_bench/benchmarks/benchmarks.py

```diff
@@ -147,6 +197,15 @@ def time_merge(self, shapes, how, sort):
             )
         )
 
+    def time_merge_2intkey(self, shapes, how, sort):
```

Comment:
Not sure I understand the name for this test. Can you elaborate?

Reply:
renamed.